easypost/constant.py

```diff
@@ -1,2 +1,5 @@
 SUPPORT_EMAIL = "support@easypost.com"
 TIMEOUT = 60
+
+FEDEX_ACCOUNT_TYPE = "FedExAccount"
+UPS_ACCOUNT_TYPE = "UpsAccount"
```

Comment:
Heh, I meant this should be a constant of type list containing the strings individually. Much easier than building multiple constants down the stack. One place of maintenance (inside the list constant) that we can then simply check against is much better than multiple single constants that we later have to build manually.

Reply:
Counter: If it's constants like this, the end-user can use them when forming their data (for example, "is it supposed to be Ups or UPS")?

Reply:
Although that would be nice, that starts to get into the weeds of "how much maintenance cost do we want to deal with" which the answer is still "as little as possible". We shouldn't expose something unless we are prepared to maintain it. If the constant name is clear we are using it for internal validation for this function, it will dissuade users from using it.

Reply:
We'd have to maintain it if the API ever changes anyway, this at least kills two birds with one stone?

Reply:
Per discussion elsewhere, this will be tweaked